src/tseda/vpages/stats.py

```diff
@@ -218,11 +216,18 @@ def __panel__(self):
         windows = make_windows(self.window_size, tsm.ts.sequence_length)
         comparisons = eval_comparisons(self.comparisons.value)
 
-        sample_sets_dictionary = self.datastore.individuals_table.sample_sets()
-        sample_sets_ids = list(sample_sets_dictionary.keys())
-        if len(sample_sets_ids) < 2:
+        selected_sample_sets = self.datastore.individuals_table.sample_sets()
+        selected_sample_sets_ids = list(selected_sample_sets.keys())
+        if len(selected_sample_sets_ids) < 2:
```

Comment:
Something seems to happen with this comparison that disables pairwise plots for more than two comparisons. In the screenshot I have selected all possible pairwise comparisons, yet only two of them show up in the plot. To reproduce you need to make a new sample set and transfer individuals in population 0 and 1 to the new sample set (id=6). 

![screen](https://github.com/user-attachments/assets/3faabe59-23ed-4dde-9d0e-2721e6216c3b)

Reply:
I don't think that the problem was the number of added comparisons. But rather that you couldn't add comparisons with the sample set ID 5 after you had merged two sample sets (and there were only 5 sample sets in total instead of the original 6). This means that there was an index-out-of-bounds issue as there is no element with index 5 in a list of only 5 elements.

This should be fixed now

Reply:
see my comments below for how it was fixed, maybe not the nicest solution...